x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/plugin/ComputeService.java

```diff
@@ -395,6 +401,8 @@ private void startComputeOnRemoteClusters(
         var linkExchangeListeners = ActionListener.releaseAfter(computeListener.acquireAvoid(), exchangeSource.addEmptySink());
         try (RefCountingListener refs = new RefCountingListener(linkExchangeListeners)) {
             for (RemoteCluster cluster : clusters) {
+                final String clusterAlias = cluster.clusterAlias();
+                final boolean suppressRemoteFailure = computeListener.shouldIgnoreRemoteErrors(clusterAlias);
                 final var childSessionId = newChildSession(sessionId);
                 ExchangeService.openExchange(
```

Comment:
We also need to ignore errors when opening an exchange.

Reply:
Added, thanks!